docs/topics/braintree.rst

```diff
@@ -57,3 +57,46 @@ Braintree already, it is not created.
 
     :status 200: customer and buyer already exist.
     :status 201: customer or buyer successfully created.
+
+Create a payment method
+-----------------------
+
+Creates a payment method in Braintree and the corresponding payment method in
+solitude.
+
+.. http:post:: /braintree/method/
+
+    :<json string uuid: the uuid of the buyer in solitude.
+    :<json string nonce: the payment nonce returned by Braintree.
+
+    Returns :ref:`a payment method object <payment-methods>` with some extra
+    fields.
+
+    .. code-block:: json
+
+        {
+            "active": true,
+            "braintree": {
+                "created_at": "2015-04-30T19:03:46.417",
+                "token": "da-token",
+                "updated_at": "2015-04-30T19:03:46.417"
+            },
+            "buyer": "/generic/buyer/13/",
+            "counter": 0,
+            "created": "2015-04-30T19:03:46.424",
+            "id": 1,
+            "modified": "2015-04-30T19:03:46.424",
+            "provider": 4,
+            "provider_id": "da-token",
+            "resource_pk": 1,
+            "resource_uri": "/generic/method/1/",
+            "truncated_id": "7890",
+            "type": 1,
+            "type_name": "visa"
+        }
+
+   :>json string braintree token: id of the payment method in braintree.
```

Comment:
I've been listing nested fields like ``braintree.token`` since that would be valid JavaScript. I wish there was a better way to display nested fields though.